src/routes/api/index.js

```diff
@@ -0,0 +1,8 @@
+const KoaRouter = require('koa-router');
+const authors = require('./authors');
+
+const router = new KoaRouter();
```

Comment:
Siguiendo con mi comentario anterior de referenciar estas rutas en `src/app.js`, habría que agregar el prefix aquí:

```suggestion
const router = new KoaRouter({ prefix: '/api' });
```